x/staking/keeper/grpc_query.go

```diff
@@ -551,6 +551,7 @@ func DelegationToDelegationResponse(ctx sdk.Context, k *Keeper, del types.Delega
 		delegatorAddress,
 		del.GetValidatorAddr(),
 		del.Shares,
+		false,
```

Comment:
Why is this false and not `del.ValidatorBond`?

Reply:
Good catch!